queries.sql

```diff
@@ -19,4 +19,64 @@ SELECT * from animals WHERE name != 'Gabumon';
 
 SELECT *
 FROM animals
-WHERE weight_kg BETWEEN 10.4 AND 17.3;
\ No newline at end of file
+WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+
+
+UPDATE animals SET species = 'unspecified';
+
```

Comment:
- [ ] You are supposed to put this inside a transaction

Reply:
Hi @descholar-ceo ,
Thanks for the feedback, i'll do that.